app/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    compileSdk 32
+
+    defaultConfig {
+        applicationId "com.parasoft.demoapp"
+        minSdk 16
+        targetSdk 32
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    sourceSets {
+        main {
+            res.srcDirs = [
+                    // Add layout files path
```

Comment:
For manage layout file better , gradle will scan each path here and import layout file.

Reply:
Organizing layout files into subfolders is good when there're many layout files. However, this may not be well supported by Android Studio,  the `layout` directory shows error underline in my Project tree. See [Organizing your Source Files | CodePath Android Cliffnotes](https://guides.codepath.com/android/Organizing-your-Source-Files#organizing-resources-into-subfolders) for other possible limitations.

Reply:
Update: The `layout` directory shows error underline in project tree problem has been fixed.